doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -467,23 +467,68 @@ destroy all backtrack and return values.
 Backtracking
 ++++++++++++
 
-In Ltac2, we have the following backtracking primitives, defined in the
-`Control` module::
+Ltac2 models backtracking computations using streams of values: the head of a non-empty stream represent the first path of computation while its tail stands for the alternative paths of computation when backtracking occurs. The empty stream holding no values represent failure and initiates backtracking.
+
+Backtracking failures (empty streams) are moreover decorated with exceptions to inform the reason of failure. These exceptions are then passed to the tail of a non-empty backtracking computation. The following Ltac2 type summarizes the model for backtracking computations
```

Comment:
```suggestion
Backtracking failures (empty streams) are decorated with exceptions that give the cause of the failure. These are passed to the tail of a non-empty backtracking computation. The following Ltac2 type summarizes the model for backtracking computations:
```

"These are passed to the tail of a non-empty backtracking computation." - I don't understand, odd wording.  How do you pass something to a computation?